sql_queries/Update_SOOP_SST_QCFlag.sql

```diff
@@ -0,0 +1,14 @@
+SET SEARCH_PATH = marvl3, public;
+
+--MAP SOOP-SST FLAGS
+\echo 'soop-sst flag'
+
+UPDATE marvl3.spatial_subset
+SET "LONGITUDE_QC"=(SELECT CASE WHEN "LONGITUDE_QC"='Z' THEN '1' ELSE '3' END),
+SET "LATITUDE_QC"=(SELECT CASE WHEN "LATITUDE_QC"='Z' THEN '1' ELSE '3' END),
+SET "TIME_QC"=(SELECT CASE WHEN  "TIME_QC"='Z' THEN '1' ELSE '3' END),
+SET "DEPTH_QC"=(SELECT CASE WHEN "DEPTH_QC"='Z' THEN '1' ELSE '3' END),
+SET "TEMP_QC"=(SELECT CASE WHEN CASE "TEMP_QC"='Z' THEN '1' ELSE '3' END),
+SET "PSAL_QC"=(SELECT CASE WHEN CASE "PSAL_QC"='Z' THEN '1' ELSE '3' END),
+WHERE source_id IN(21,22);
```

Comment:
We don't need this file to be in the repo. Thanks for sharing it so that we can fix our table without having to reload the impacted collection but this is not relevant to be present in the repo in my opinion.